src/backend/lima.ts

```diff
@@ -1809,22 +1838,29 @@ class LimaKubernetesBackend extends events.EventEmitter implements K8s.Kubernete
     });
 
     try {
+      const persistedVersion = await this.getPersistedVersion();
       const desiredVersion = await this.desiredVersion;
+      const isDowngrade = (version: semver.SemVer | string) => {
```

Comment:
I think for this function it was better to have it local (so that we don't end up accidentally using it outside the function); it's not really general.

Also, I'm capturing the value of `persistedVersion`, so a (global?) function would need to reworked.

Reply:
Yeah fair enough. Let's get this merged :smiley: